### PR TITLE
Enhancing documentation for missing configuration

### LINKED ISSUE #131: Describe all configuration options
Describe all configuration options on the "Getting Started" page. Right now only `base_url` is described. Next to each option a `(since vX.Y.Z)` needs to be added for each option added since 1.0.0 release.

### PR DESCRIPTION
**Pull request by evangelion1204**
A basic idea for documenting `page_namespace_prefix` and page locators.

Closes #131

### CODE REVIEW

File: docs/getting-started.rst [hunk 1]
```diff
@@ -53,6 +53,8 @@ Configuration Options
 The following configuration options are available:
 
 * ``base_url`` - allows to specify Base URL to be used to transform all relative urls from :ref:`@page-url annotations <page-url>` into absolute urls.
+* ``page_namespace_prefix`` - array of namespaces in which the :ref:`DefaultPageLocator <page-locator>` will search for page classes defaulted with ``array('\\')``
```
Comment (aik099) [inline docs/getting-started.rst]: I don't know if word `defaulted` exists, but `defaults to` sounds more human friendly.

File: docs/glossary.rst [hunk 2]
```diff
@@ -21,3 +21,63 @@ Annotation
...
 ------------
 ...
+
+
+.. _page-locator:
+
+Page Locator
+------------
+
+The page locator is used by the page factory to locate/resolve the FQCN defined by a given name.
```
Comment (aik099) [inline docs/glossary.rst]: If I wasn't aware, that page locator is transforming page names into page classes where these pages are implemented, then I never could have guessed by this text. This sentence needs to be changed seriously.

File: docs/glossary.rst [hunk 3]
```diff
@@ -21,3 +21,63 @@ Annotation
 Page Factory
 ------------
 ...
+
+
+.. _page-locator:
+
+Page Locator
```
Comment (aik099) [inline docs/glossary.rst]: I like the information, but it's placed on incorrect page. The glossary page is for describing concepts used in library without any real code example or similar information. It's ok to create new page for describing page locator specifically.

File: docs/examples/custom_page_locator.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?php
+use QATools\QATools\PageObject\Exception\PageFactoryException;
+use QATools\QATools\PageObject\PageLocator\IPageLocator;
+
+class MappingPageLocator implements IPageLocator
+{
+	protected $pages = array(
```
Comment (aik099) [inline docs/examples/custom_page_locator.php]: Unfortunately with TABs used for indentation the code isn't formatted properly and therefore we need to replace them with 4 spaces.

**Comment by evangelion1204** [comment]
Moved `DefaultPageLocator` and Custom Locator to new how-to page. Changed text for locator/url and format of PHP file.

File: docs/glossary.rst (hunk 3 above)
Comment (evangelion1204) [inline docs/glossary.rst]: Moved it to a new page.

**Comment by aik099** [comment]
Please fix RST issue reported by Travis.

**Comment by evangelion1204** [comment]
ok, corrected :smile:

**Comment by aik099** [comment]
Sudden coverage drop (-36%) is caused by ParaTest (see https://github.com/brianium/paratest/issues/160). Maybe we should stop using paratest at all. Not sure if this would slow down test suite much.

**Comment by aik099** [comment]
Thanks, merging.

**Pull request merged**